spec/features/cco/dashboard_spec.rb

```diff
@@ -4,85 +4,62 @@
+      cco_user = create :cco_user
+      aborted_defence_request = create :defence_request, :aborted
+      acknowledged_defence_request = create :defence_request, :acknowledged
+      queued_defence_request = create :defence_request, :queued
 
-    before :each do
-      create_role_and_login("cco")
-    end
-
-    specify 'can see tables of "queued", "acknowledged" and "aborted" defence requests' do
-      visit defence_requests_path
-
-      within ".queued-defence-request" do
-        expect(page).to have_content(dr_queued.solicitor_name)
-      end
+      login_with_role "cco", cco_user.uid
 
-      within ".acknowledged-defence-request" do
-        expect(page).to have_content(dr_ack.solicitor_name)
-      end
-
-      within ".aborted-defence-request" do
-        expect(page).to have_content(dr_aborted.solicitor_name)
-      end
+      expect(page).to have_content aborted_defence_request.solicitor_name
+      expect(page).to have_content acknowledged_defence_request.solicitor_name
+      expect(page).to have_content queued_defence_request.solicitor_name
     end
 
-    specify "can see the dashboard with refreshed data after a period", js: true, short_dashboard_refresh: true do
-      visit defence_requests_path
-
-      within "#defence_request_#{dr_queued.id}" do
-        expect(page).to have_content(dr_queued.solicitor_name)
-      end
+    specify "view refreshed data on the dashboard with after a period", js: true, short_dashboard_refresh: true do
+      cco_user = create :cco_user
+      acknowledged_defence_request = create :defence_request, :acknowledged
+      queued_defence_request = create :defence_request, :queued
 
-      within "#defence_request_#{dr_ack.id}" do
-        expect(page).to have_content(dr_ack.solicitor_name)
-      end
+      login_with_role "cco", cco_user.uid
 
-      dr_queued.update(solicitor_name: "New Solicitor")
-      dr_ack.update(solicitor_name: "New Solicitor2")
+      acknowledged_defence_request.update(
+        solicitor_name: "Queued Req. Updated Solicitor Name"
+      )
+      queued_defence_request.update(
+        solicitor_name: "Acknowledged Req. Updated Solicitor Name"
+      )
 
       wait_for_dashboard_refresh
 
-      within "#defence_request_#{dr_queued.id}" do
-        expect(page).to have_content("New Solicitor")
-      end
-      within "#defence_request_#{dr_ack.id}" do
-        expect(page).to have_content("New Solicitor2")
-      end
-
+      expect(page).to have_content "Queued Req. Updated Solicitor Name"
+      expect(page).to have_content "Acknowledged Req. Updated Solicitor Name"
     end
 
-    specify "can resend case details of an defence request to the assigned solicitor", js: true do
-      visit defence_requests_path
-
-      within ".accepted-defence-request" do
-        click_button "Resend details"
-      end
+    xspecify "can resend case details of an defence request to the assigned solicitor", js: true do
```

Comment:
Yes, after discussion it was decided to have these specs pending, there are about 17 in total I believe.